config/logicsoft/ZDB5100.xml

```diff
@@ -0,0 +1,498 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product Revision="2" xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="Name">Matrix</MetaDataItem>
+    <MetaDataItem name="Description">MATRIX ZDB5100 is a dimmer wall switch module with four backlit pushbuttons. MATRIX is designed to fit into 1-module FUGA® wall boxes (Danish standard).</MetaDataItem>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/0234:0121:0003</MetaDataItem>
+    <MetaDataItem name="ProductPage">https://logic-group.com/products/matrix-switches/</MetaDataItem>
+    <MetaDataItem name="ProductSupport">https://logic-group.com/contact/</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/logicsoft/ZDB5100.png</MetaDataItem>
```

Comment:
please also add this file to your PR please

Reply:
Thanks. I must have been tired. This will be added with next commit.